docs/quick-start.md

```diff
@@ -299,19 +299,17 @@ We'll create a very simple Spark application, `SimpleApp.java`:
 import org.apache.spark.sql.SparkSession;
 import org.apache.spark.sql.Dataset;
 
-public class SimpleApp {
-  public static void main(String[] args) {
-    String logFile = "YOUR_SPARK_HOME/README.md"; // Should be some file on your system
-    SparkSession spark = SparkSession.builder().appName("Simple Application").getOrCreate();
-    Dataset<String> logData = spark.read().textFile(logFile).cache();
+public class SimpleApp extends App {
```

Comment:
Yes, this isn't valid Java, and it's supposed to be Java. It was correct before.

Reply:
Sorry, I updated the wrong section.